modules/passkey/test/libraries/P256.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('P256', function () {
   it('Should return true on valid signature', async function () {
     const { verifier, p256Lib, account } = await setupTests()
 
-    const message = ethers.id('hello world')
+    const message = ethers.id('hello passkeys')
```

Comment:
I accidentally opened this file instead of WebAuthn tests and changed `world` to `passkeys` for fun, and that led to discovering https://github.com/safe-global/safe-modules/pull/338. I can revert the change but IMO, it looks more fun like this :stuck_out_tongue: